openedx/core/djangoapps/announcements/templates/announcements/announcements_edit.html

```diff
@@ -0,0 +1,41 @@
+{% load static %}
+
+{% block title %}{{_("Edit Announcement")}}{% endblock %}
+
+{% block content %}
+<div class="wrapper-content wrapper announcements-editor">
```

Comment:
Would be much nicer/cleaner if the adding/editing could be done inline on the list page, without having to load another page.